lib/saml/user.rb

```diff
@@ -17,35 +17,37 @@ def last_signed_in
       Time.current.utc
     end
 
-    def loa
-      { current: loa_current, highest: loa_highest(attributes) }
-    end
```

Comment:
Starting to think `def loa` should be removed entirely, not only from `user`, but also from IDme, DSlogon, & MHV decorators.  Then in the `user_serializer`, make it read:

```
loa: { current: object.loa_current, highest: object.loa_highest }
```

This isn't super important, but helps to ensure we don't break the existing API contract provided by the `GET /v0/user` endpoint.  The whole reason I'm commenting on this line at all, was because I was scared one of the IDme, DSlogon, or MHV decorators broke said contract (but they didn't :smile: ).

Reply:
I was trying to be very mindful of this fact, especially after learning that most of the apps on the F/E got lazy and started gating on loa3 directly vs going by the services listed in user_serializer.